index.html

```diff
@@ -147,8 +176,9 @@ <h5 class="card-heading heading-color">Skills</h5>
 </main>
 
 <footer class="footing background">
+    <div class="footer-main-container">
 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-<div>
+<div class="footer-right-container">
 <form class="form-container user-form" action="https://formspree.io/f/xbjqyrpj" method="POST">  
     <input type="text" class="user-info name" placeholder="Full name" name="user-name" maxlength="30" required>
```

Comment:
- [ ] For the desktop version you need to add an extra input, one for your first name and the other for your last name

**Your design:**

![image](https://user-images.githubusercontent.com/60631456/121705254-13268d00-ca9a-11eb-8a60-a5cd0f0f9fe3.png)

**Original design:**

![image](https://user-images.githubusercontent.com/60631456/121705363-2cc7d480-ca9a-11eb-8014-07481f6b0d14.png)